src/activity_handlers.cpp

```diff
@@ -955,9 +956,10 @@ void activity_handlers::forage_finish( player_activity *act, player *p )
         }
     }
 
-    if( rng( 1, 5 ) > 1 ) {
-        g->m.spawn_item( p->pos(), "withered", rng(1, 3) );
-        found_something = true;
+	if( bush_chance < p->get_skill_level( skill_survival) * 3 + p->per_cur -2 && !found_something) {
+        g->m.spawn_item(p->pos(), "withered", rng(1, 3)); 
+        add_msg(m_good, _("You found: %s!" ), "withered plant" ); 
```

Comment:
Your version still has tabs in it. You're supposed to intend with 4 spaces per level.

The seconds string here isn't translated. Why is it a separate string anyway? Just print one string with translation (the `_()`).

Reply:
Sorry about the tabs I think vim is automatically adding them even tho I have setup in my rules to not use them.... Ill have to check my config again. I did that cause thats how the other messages were being printed and the previous pr did not add the notification message that anything was found. I was unsure what the elements that were needed for add_msg were or if it could accept only 2 arguments. Sorry I am more familiar with C not C++ and even then its been a long while.